src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Leaderboard</title>
+</head>
+<body>
+  <h1 id="leader">Leaderboard</h1>
+  <div class="main-section">
+    <div class="scores-section">
+    <div class="heading">
+      <h2>Recent scores</h2>
+      <div id="btn-refresh" class="refresh"><button type="submit" class="refresh">Refresh</button></div>
+    </div>
+    <div id="list" class="list"></div>
+    </div>
+      <div class="leader-board" id="leader-board">
+        <h2>Add your score</h2>
+        <form name="board" id="form" class="form" action="submit">
+          <input type="text" id="name" name="name" placeholder="Your name"><br><br>
+          <input type="text" id="score" name="score" placeholder="Your score"><br><br>
+          <button type="submit" id="btn" class="btn">Submit</button>
+        </form>
```

Comment:
[OPTIONAL] Name and score fields should be properly validated to ensure a user cannot submit blank entries. Ensure that error message is displayed when a user tries to submit empty fields.